CONTRIBUTING.md

```diff
@@ -5,15 +5,17 @@ First of all, thank you for contributing. It’s appreciated.
 1. Open a GitHub issue before doing significant amount of work.
 2. Clone the repo. If it was already cloned, then git pull to get the latest from master.
 3. Use a Linux or Mac computer. No Windows, sorry.
-4. Ensure Yarn is [installed](https://yarnpkg.com/lang/en/docs/install/).
-5. Run `make setup` before anything else, and wait.
+4. Ensure pnpm is [installed](https://pnpm.js.org/docs/en/installation.html).
+5. Run `pnpm recursive install` before anything else, and wait.
 6. Write code.
-7. Run `make test dom` (replace `dom` with the package you are testing) to lint and test. Don’t commit before fixing all errors and warnings.
-8. Commit using `make commit` and follow the CLI instructions.
+7. Run `pnpm recursive test --scope @cycle/dom` (replace `dom` with the package you are testing) to lint and test. Don’t commit before fixing all errors and warnings.
```

Comment:
`pnpm recursive test --scope @cycle/dom` is quite long, could we recommend something else or make an alias? Like either `cd dom && pnpm test` or `pnpm run test-dom` or something else

Reply:
will do `cd dom && pnpm test`